sqlmesh/core/model/definition.py

```diff
@@ -2034,14 +2035,22 @@ def _create_model(
     )
 
     dialect = dialect or ""
-    physical_schema_override = physical_schema_override or {}
+    physical_schema_mapping = physical_schema_mapping or {}
 
     raw_kind = kwargs.pop("kind", None)
     if raw_kind:
         kwargs["kind"] = create_model_kind(raw_kind, dialect, defaults or {})
 
     defaults = {k: v for k, v in (defaults or {}).items() if k in klass.all_fields()}
 
+    def _resolve_physical_schema_override(schema_name: str) -> t.Optional[str]:
```

Comment:
I just moved the resolving logic to here now that it only depends on a single property

Reply:
And then I changed my mind and removed the function altogether to just do this in line